uploader/README.md

```diff
@@ -0,0 +1,130 @@
+# Video Upload Pipeline
+
+**NB: this describes a work in-progress**
+
+The direct video upload pipeline is implemented using AWS step functions co-ordinating
+a number of lambdas. It has its own cloud-formation stack (`media-atom-pipeline`) that
+is deployed automatically by Riff-Raff alongside the main `media-atom-maker` app. Each
+lambda has a separate entry point into a single Scala code bundle (the `uploader` SBT
+project).
+
+The cloud-formation is assembled from a series of templates in the `uploader` project
+during packaging.
+
+## Deploying the DEV stack
+
+You will need the AWS CLI and `jq` installed.
+
+Run:
+
+```
+uploader/packageBin
+```
+
+This will compile both the cloud-formation template and the lambda code.
+
+The cloud-formation template will be output at:
+
+```
+uploader/target/scala-2.11/resource_managed/main/media-atom-pipeline.yaml
+```
+
+Use the AWS console to update the DEV stack with the new template. Unfortunately this
+cannot be scripted as federated Janus credentials cannot modify IAM roles 😞.
```

Comment:
Not sure what you mean? Anyone who has developer credentials for the account will be able to do it from the web console.